### PR TITLE
Update Darwinia Chain docs II

### PR DESCRIPTION
**Pull request by boundless-forest**

### CODE REVIEW

File: docs/chains/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 0
+title: Introduction
+---
+
+Darwinia offers a number of different chains for development of messaging infrastructure and Dapp deployment, each with its own features and benefits.
+
+## [Darwinia Chain](darwinia-chain.md)
+
+Darwinia Chain(DC) is designed to be the primary chain for tokens and governance, main communication hub and routing point of the LCMP, and will support most on-chain light clients for other public chains.
```
Comment (wuminzhe) [inline docs/chains/Introduction.md]: Should we remove the short forms? We almost never use it.

Comment (boundless-forest) [reply docs/chains/Introduction.md]: Just to confirm, do the short forms above mean `DC` or `DP`?

File: docs/chains/darwinia-smart-chain.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ title: Darwinia Smart Chain
 Darwinia Smart Chain is a substrate based ethereum compatible smart contract platform where contract developers can easily migrate
 smart contracts from the Ethereum chain to run their business. In this article we will cover a number of topics that are important to contract developers, such as network configuration, pre-compiles and so on. In addition, we have set up a free test network, the pangoro network, where the latest features will be fully tested before they are added to the Darwinia Smart Chain. The user can obtain test tokens at [Pangoro Test Token Faucet](https://apps.darwinia.network/?network=pangoro).
```
Comment (wuminzhe) [inline docs/chains/darwinia-smart-chain.md]: `precompiles` maybe better.

`precompiles` or `precompiled contracts` is a convention name in the evm world.

File: docs/chains/Introduction.md (hunk 1 above)
Comment (wuminzhe) [reply docs/chains/Introduction.md]: DC for Darwinia Chain

File: docs/chains/darwinia-smart-chain.md (hunk 2 above)
Comment (boundless-forest) [reply docs/chains/darwinia-smart-chain.md]: done

**Review by wuminzhe — APPROVED** [review]

**Pull request merged**